apps/dashboard/app/apps/support_ticket_rt_service.rb

```diff
@@ -12,7 +20,7 @@ class SupportTicketRtService
   #
   # @return [SupportTicket] support_ticket model
   def default_support_ticket(request_params)
-    support_ticket = SupportTicket.from_config(::Configuration.support_ticket_config)
+    support_ticket = SupportTicket.from_config(@support_ticket_config)
```

Comment:
Can we try to use `attr_reader` for stuff like this? I just feel like that's the patter, to use them instead of directly referencing instance variables.

Reply:
fixed